packages/core/src/common/_font-imports.scss

```diff
@@ -3,11 +3,11 @@
 // of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
 // and https://github.com/palantir/blueprint/blob/master/PATENTS
 
-@import "~bourbon/app/assets/stylesheets/bourbon";
-@import "variables";
+@import '~bourbon/app/assets/stylesheets/bourbon';
+@import 'variables';
```

Comment:
we're all about double quotes over here. gonna fail stylelint until you revert all these quote changes.